functions/lib/integration/import-order.js

```diff
@@ -103,7 +103,10 @@ module.exports = ({ appSdk, storeId, auth }, tinyToken, queueEntry, appData, can
   if (typeof tinyOrderNumber === 'string' && tinyOrderNumber.startsWith('id:')) {
     job = getTinyOrder(tinyOrderNumber.substring(3))
   } else {
-    job = tiny.post('/pedidos.pesquisa.php', { numero: tinyOrderNumber })
+    const filter = typeof tinyOrderNumber === 'string' && tinyOrderNumber.startsWith('ecom:')
+      ? { numeroEcommerce: String(tinyOrderNumber) }
```

Comment:
```suggestion
      ? { numeroEcommerce: String(tinyOrderNumber).substring(5) }
```

corrigindo minha lerdeza mesmo 🙃